examples/context/theme-detailed-themed-button.js

```diff
@@ -0,0 +1,19 @@
+import {ThemeContext} from './theme-context';
+
+class ThemedButton extends React.Component {
```

Comment:
This could be a functional component, right? I think maybe we should use functional components in cases where a class isn't necessary.